test/pod.bats

```diff
@@ -336,7 +336,7 @@ function teardown() {
 
 @test "kubernetes pod terminationGracePeriod passthru" {
 	[ -v CIRCLECI ] && skip "runc v1.0.0-rc11 required" # TODO remove this
-	CONTAINER_CGROUP_MANAGER="systemd" start_crio
+	CONTAINER_CGROUP_MANAGER="systemd" XDG_RUNTIME_DIR="" start_crio
```

Comment:
I think we should have an explaination on why we need this change in a comment

Reply:
sure, added